lib/automate/__init__.py

```diff
@@ -70,22 +71,32 @@ def prepare(self, module_name, text):
         :type text: string
         :rtype: string
         """
-        sender = SETTINGS["user"]
+
+        spin = spinner.Spinner()
```

Comment:
Not sure what is best here but spinners will be executed even when using the GUI. It might be useful for the GUI to know when something is loading in automation as well so maybe we should work with events or something? Just a thought

Reply:
Nice observation, didn't even think about that. The GUI surely does not desire this code to run in the background. I'll take a look and see if I can solve it differently.